hack/verify-generated-crd.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+TMP_DIR="$(mktemp -d)"
+
+OUTDIR=$TMP_DIR SKIP_COPY=true ./hack/update-generated-crd.sh
+
+diff -Naup $TMP_DIR/operator.openshift.io_ingresscontrollers.yaml manifests/00-custom-resource-definition.yaml
```

Comment:
Better to quote to avoid word splitting on `$TMP_DIR`.

Reply:
Fixed